2024-08-23 12:23:19 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00804517244157992538389ed211","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:23:19 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:23:22 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 12:23:22 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00804517244158025208753ed211","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:23:22 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:23:26 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 12:23:26 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 12:28:28 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /QOA7NlfR%2F889478%2F6685379767043d4423c0406286920ffb5ca4427a%2F66c8804e7b911d9159164e069f1f62156ef68046?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DNBsDNpudb57ioUkPJvFobTfZhl7Xf1zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNQqBW0%2Bnq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneY5VN0QO8i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iftbyCPXtuMkagAERjL0fRc5kSGCUb6vQagjV56KN6MpB0b%2B1QEHnh4wb8vlU9gpplp5oWboIp437fVCoFng7zDPvnHZIjyEE8rvapcq%2BOw%2FmWhBb7GBf6n4cmbKXqPiFGXfall1tNOapmndqcTfJK1sSzNvTn7XVJ9aMdBGCQaY6pr7zmqBBqgg1lyAA&uploadId=996ECF5A8D864CEDB945CEC7AB18ACB6&x-oss-access-key-id=STS.NSxMzyK8zPzCPexXZNAyMxpt5&x-oss-expires=1724419678&x-oss-signature=4BiY3K4r10qcuzd6ASkgPz2%2BCn8O6yRX27qHABOwAdo%3D&x-oss-signature-version=OSS2
2024-08-23 12:29:00 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /QOA7NlfR%2F889478%2F6685379767043d4423c0406286920ffb5ca4427a%2F66c8804e7b911d9159164e069f1f62156ef68046?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DNBsDNpudb57ioUkPJvFobTfZhl7Xf1zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNQqBW0%2Bnq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneY5VN0QO8i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iftbyCPXtuMkagAERjL0fRc5kSGCUb6vQagjV56KN6MpB0b%2B1QEHnh4wb8vlU9gpplp5oWboIp437fVCoFng7zDPvnHZIjyEE8rvapcq%2BOw%2FmWhBb7GBf6n4cmbKXqPiFGXfall1tNOapmndqcTfJK1sSzNvTn7XVJ9aMdBGCQaY6pr7zmqBBqgg1lyAA&uploadId=996ECF5A8D864CEDB945CEC7AB18ACB6&x-oss-access-key-id=STS.NSxMzyK8zPzCPexXZNAyMxpt5&x-oss-expires=1724419678&x-oss-signature=4BiY3K4r10qcuzd6ASkgPz2%2BCn8O6yRX27qHABOwAdo%3D&x-oss-signature-version=OSS2
2024-08-23 12:30:37 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244162372276373ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:30:37 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:30:38 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 12:30:38 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244162384856522ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:30:38 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:30:40 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 12:30:40 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244162407426750ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:30:40 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:30:43 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 12:30:44 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244162440017168ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:30:44 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:30:48 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 12:30:48 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 12:30:58 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244162580988821ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:30:58 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:30:59 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 12:30:59 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244162593688990ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:30:59 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:31:01 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 12:31:01 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244162616451266ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:31:01 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:31:04 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 12:31:05 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244162649031695ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:31:05 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:31:09 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 12:31:09 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 12:33:32 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /n3ke0yYp%2F889478%2F66a0ddc039398f6536d64ac19c22d47862b56b15%2F66c8817dd146635f16734403b24254f8321c76c9?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DNBsDNpudb57ioUkPJvFobTfZhl7Xf1zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNQqBW0%2Bnq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneY5VN0QO8i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iftbyCPXtuMkagAERjL0fRc5kSGCUb6vQagjV56KN6MpB0b%2B1QEHnh4wb8vlU9gpplp5oWboIp437fVCoFng7zDPvnHZIjyEE8rvapcq%2BOw%2FmWhBb7GBf6n4cmbKXqPiFGXfall1tNOapmndqcTfJK1sSzNvTn7XVJ9aMdBGCQaY6pr7zmqBBqgg1lyAA&uploadId=CFF8CC5569914F4288437C660A26A708&x-oss-access-key-id=STS.NSxMzyK8zPzCPexXZNAyMxpt5&x-oss-expires=1724419981&x-oss-signature=iLT20wIr5iIQrA04htmPvuwCuPAaGmZb8BPn6OmhJTY%3D&x-oss-signature-version=OSS2
2024-08-23 12:34:03 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /n3ke0yYp%2F889478%2F66a0ddc039398f6536d64ac19c22d47862b56b15%2F66c8817dd146635f16734403b24254f8321c76c9?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DNBsDNpudb57ioUkPJvFobTfZhl7Xf1zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNQqBW0%2Bnq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneY5VN0QO8i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iftbyCPXtuMkagAERjL0fRc5kSGCUb6vQagjV56KN6MpB0b%2B1QEHnh4wb8vlU9gpplp5oWboIp437fVCoFng7zDPvnHZIjyEE8rvapcq%2BOw%2FmWhBb7GBf6n4cmbKXqPiFGXfall1tNOapmndqcTfJK1sSzNvTn7XVJ9aMdBGCQaY6pr7zmqBBqgg1lyAA&uploadId=CFF8CC5569914F4288437C660A26A708&x-oss-access-key-id=STS.NSxMzyK8zPzCPexXZNAyMxpt5&x-oss-expires=1724419981&x-oss-signature=iLT20wIr5iIQrA04htmPvuwCuPAaGmZb8BPn6OmhJTY%3D&x-oss-signature-version=OSS2
2024-08-23 12:34:34 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /n3ke0yYp%2F889478%2F66a0ddc039398f6536d64ac19c22d47862b56b15%2F66c8817dd146635f16734403b24254f8321c76c9?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DNBsDNpudb57ioUkPJvFobTfZhl7Xf1zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNQqBW0%2Bnq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneY5VN0QO8i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iftbyCPXtuMkagAERjL0fRc5kSGCUb6vQagjV56KN6MpB0b%2B1QEHnh4wb8vlU9gpplp5oWboIp437fVCoFng7zDPvnHZIjyEE8rvapcq%2BOw%2FmWhBb7GBf6n4cmbKXqPiFGXfall1tNOapmndqcTfJK1sSzNvTn7XVJ9aMdBGCQaY6pr7zmqBBqgg1lyAA&uploadId=CFF8CC5569914F4288437C660A26A708&x-oss-access-key-id=STS.NSxMzyK8zPzCPexXZNAyMxpt5&x-oss-expires=1724419981&x-oss-signature=iLT20wIr5iIQrA04htmPvuwCuPAaGmZb8BPn6OmhJTY%3D&x-oss-signature-version=OSS2
2024-08-23 12:35:44 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='cn-beijing-data.aliyundrive.net', port=443): Read timed out. (read timeout=30)")': /Y8dRz4GA%2F889478%2F6672db37c6715351df9c41ed9868969af2a24bdf%2F66c881efcc915078af74490596d56b55a56b7a10?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DNBsDNpudb57ioUkPJvFobTfZhl7Xf1zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNQqBW0%2Bnq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneY5VN0QO8i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iftbyCPXtuMkagAERjL0fRc5kSGCUb6vQagjV56KN6MpB0b%2B1QEHnh4wb8vlU9gpplp5oWboIp437fVCoFng7zDPvnHZIjyEE8rvapcq%2BOw%2FmWhBb7GBf6n4cmbKXqPiFGXfall1tNOapmndqcTfJK1sSzNvTn7XVJ9aMdBGCQaY6pr7zmqBBqgg1lyAA&uploadId=8511A8E93D044E0C90FFF47312651FE2&x-oss-access-key-id=STS.NSxMzyK8zPzCPexXZNAyMxpt5&x-oss-expires=1724420095&x-oss-signature=wNcyvgOCGa28EqjpUHRQ95v7pNzAvTh1pW7mqT%2BRhrA%3D&x-oss-signature-version=OSS2
2024-08-23 12:36:15 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /Y8dRz4GA%2F889478%2F6672db37c6715351df9c41ed9868969af2a24bdf%2F66c881efcc915078af74490596d56b55a56b7a10?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DNBsDNpudb57ioUkPJvFobTfZhl7Xf1zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNQqBW0%2Bnq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneY5VN0QO8i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iftbyCPXtuMkagAERjL0fRc5kSGCUb6vQagjV56KN6MpB0b%2B1QEHnh4wb8vlU9gpplp5oWboIp437fVCoFng7zDPvnHZIjyEE8rvapcq%2BOw%2FmWhBb7GBf6n4cmbKXqPiFGXfall1tNOapmndqcTfJK1sSzNvTn7XVJ9aMdBGCQaY6pr7zmqBBqgg1lyAA&uploadId=8511A8E93D044E0C90FFF47312651FE2&x-oss-access-key-id=STS.NSxMzyK8zPzCPexXZNAyMxpt5&x-oss-expires=1724420095&x-oss-signature=wNcyvgOCGa28EqjpUHRQ95v7pNzAvTh1pW7mqT%2BRhrA%3D&x-oss-signature-version=OSS2
2024-08-23 12:36:18 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/Y8dRz4GA%2F889478%2F6672db37c6715351df9c41ed9868969af2a24bdf%2F66c881efcc915078af74490596d56b55a56b7a10?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DNBsDNpudb57ioUkPJvFobTfZhl7Xf1zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNQqBW0%2Bnq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneY5VN0QO8i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iftbyCPXtuMkagAERjL0fRc5kSGCUb6vQagjV56KN6MpB0b%2B1QEHnh4wb8vlU9gpplp5oWboIp437fVCoFng7zDPvnHZIjyEE8rvapcq%2BOw%2FmWhBb7GBf6n4cmbKXqPiFGXfall1tNOapmndqcTfJK1sSzNvTn7XVJ9aMdBGCQaY6pr7zmqBBqgg1lyAA&uploadId=8511A8E93D044E0C90FFF47312651FE2&x-oss-access-key-id=STS.NSxMzyK8zPzCPexXZNAyMxpt5&x-oss-expires=1724420095&x-oss-signature=wNcyvgOCGa28EqjpUHRQ95v7pNzAvTh1pW7mqT%2BRhrA%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>66C88240D46E713131A4CBA7</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"988B73EE3F14457C6091A630317B4DCC"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-08-23 12:36:20 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/Y8dRz4GA%2F889478%2F6672db37c6715351df9c41ed9868969af2a24bdf%2F66c881efcc915078af74490596d56b55a56b7a10?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DNBsDNpudb57ioUkPJvFobTfZhl7Xf1zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNQqBW0%2Bnq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneY5VN0QO8i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iftbyCPXtuMkagAERjL0fRc5kSGCUb6vQagjV56KN6MpB0b%2B1QEHnh4wb8vlU9gpplp5oWboIp437fVCoFng7zDPvnHZIjyEE8rvapcq%2BOw%2FmWhBb7GBf6n4cmbKXqPiFGXfall1tNOapmndqcTfJK1sSzNvTn7XVJ9aMdBGCQaY6pr7zmqBBqgg1lyAA&uploadId=8511A8E93D044E0C90FFF47312651FE2&x-oss-access-key-id=STS.NSxMzyK8zPzCPexXZNAyMxpt5&x-oss-expires=1724420095&x-oss-signature=wNcyvgOCGa28EqjpUHRQ95v7pNzAvTh1pW7mqT%2BRhrA%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>66C88243D8670638366BDF5C</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"988B73EE3F14457C6091A630317B4DCC"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-08-23 12:36:21 - WARNING - base.py - 100 - https://cn-beijing-data.aliyundrive.net/Y8dRz4GA%2F889478%2F6672db37c6715351df9c41ed9868969af2a24bdf%2F66c881efcc915078af74490596d56b55a56b7a10?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DNBsDNpudb57ioUkPJvFobTfZhl7Xf1zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNQqBW0%2Bnq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneY5VN0QO8i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iftbyCPXtuMkagAERjL0fRc5kSGCUb6vQagjV56KN6MpB0b%2B1QEHnh4wb8vlU9gpplp5oWboIp437fVCoFng7zDPvnHZIjyEE8rvapcq%2BOw%2FmWhBb7GBf6n4cmbKXqPiFGXfall1tNOapmndqcTfJK1sSzNvTn7XVJ9aMdBGCQaY6pr7zmqBBqgg1lyAA&uploadId=8511A8E93D044E0C90FFF47312651FE2&x-oss-access-key-id=STS.NSxMzyK8zPzCPexXZNAyMxpt5&x-oss-expires=1724420095&x-oss-signature=wNcyvgOCGa28EqjpUHRQ95v7pNzAvTh1pW7mqT%2BRhrA%3D&x-oss-signature-version=OSS2:409:<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>PartAlreadyExist</Code>
  <Message>For sequential multipart upload, you can't overwrite uploaded parts.</Message>
  <RequestId>66C88244B8F877343078FFB1</RequestId>
  <HostId>cn-beijing-data.aliyundrive.net</HostId>
  <PartEtag>"988B73EE3F14457C6091A630317B4DCC"</PartEtag>
  <PartNumber>1</PartNumber>
  <EC>0042-00000108</EC>
  <RecommendDoc>https://api.aliyun.com/troubleshoot?q=0042-00000108</RecommendDoc>
</Error>

2024-08-23 12:36:21 - ERROR - aliyundrive.py - 1077 - File tmp/119782500/119782500_p0.png Part 1 upload failed.
2024-08-23 12:39:00 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700817244167404267863e1405","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:39:00 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:39:01 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 12:39:01 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700817244167416737967e1405","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:39:01 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:39:03 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 12:39:04 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700817244167439388200e1405","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:39:04 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:39:07 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 12:39:07 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700817244167471828586e1405","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:39:07 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:39:11 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 12:39:11 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 12:39:30 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700817244167706193219e1405","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:39:30 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:39:31 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 12:39:32 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700817244167718653468e1405","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:39:32 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:39:34 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 12:39:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700817244167741163817e1405","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:39:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:39:37 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 12:39:37 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00700817244167773934187e1405","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:39:37 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:39:41 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 12:39:41 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 12:44:24 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', TimeoutError('The write operation timed out'))': /KwM1TG8d%2F889478%2F63ad84602507d65f08d14e2c9a7aa3f83b9d4068%2F66c88409c9120c3427884574b0869a61c48b86d2?partNumber=1&security-token=CAISvgJ1q6Ft5B2yfSjIr5DNBsDNpudb57ioUkPJvFobTfZhl7Xf1zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNQqBW0%2Bnq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneY5VN0QO8i4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iftbyCPXtuMkagAERjL0fRc5kSGCUb6vQagjV56KN6MpB0b%2B1QEHnh4wb8vlU9gpplp5oWboIp437fVCoFng7zDPvnHZIjyEE8rvapcq%2BOw%2FmWhBb7GBf6n4cmbKXqPiFGXfall1tNOapmndqcTfJK1sSzNvTn7XVJ9aMdBGCQaY6pr7zmqBBqgg1lyAA&uploadId=84BBF6567E9D4136BB7945D7771BA773&x-oss-access-key-id=STS.NSxMzyK8zPzCPexXZNAyMxpt5&x-oss-expires=1724420634&x-oss-signature=1h37hcUJAgo744AHYkFSULG%2FCeBTNj7o7WCFa4%2FWa94%3D&x-oss-signature-version=OSS2
2024-08-23 12:45:44 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00804517244171444013588ed21c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:45:44 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:45:45 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 12:45:45 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00804517244171456483732ed21c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:45:45 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:45:47 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 12:45:48 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00804517244171478923940ed21c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:45:48 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:45:51 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 12:45:51 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00804517244171511394369ed21c","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 12:45:51 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 12:45:55 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 12:45:55 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 22:16:48 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-08-23 22:18:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0080fc17244514985584547e0263","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:18:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:18:19 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 22:18:19 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0080fc17244514998204669e0263","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:18:19 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:18:21 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 22:18:22 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0080fc17244515020884942e0263","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:18:22 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:18:25 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 22:18:25 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0080fc17244515053505320e0263","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:18:25 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:18:29 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 22:18:29 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 22:20:26 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/07/26/00/09/35/120882628_p0.png
2024-08-23 22:22:00 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803617244517201665288ef0e7","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:22:00 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:22:01 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 22:22:01 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803617244517214495425ef0e7","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:22:01 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:22:03 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 22:22:03 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803617244517237135664ef0e7","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:22:03 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:22:06 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 22:22:07 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803617244517269756018ef0e7","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:22:07 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:22:11 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 22:22:11 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 22:23:42 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803617244518220748843ef0e7","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:23:42 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:23:43 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 22:23:43 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803617244518233381021ef0e7","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:23:43 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:23:45 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 22:23:45 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803617244518256051359ef0e7","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:23:45 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:23:48 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 22:23:48 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803617244518288721788ef0e7","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:23:48 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:23:52 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 22:23:52 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 22:27:02 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803417244520228671702e38bd","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:27:02 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:27:03 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 22:27:04 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803417244520241391897e38bd","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:27:04 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:27:06 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 22:27:06 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803417244520264062267e38bd","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:27:06 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:27:09 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 22:27:09 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803417244520296702707e38bd","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:27:09 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:27:13 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 22:27:13 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 22:32:21 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244523411586733e0264","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:32:21 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:32:22 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 22:32:22 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244523424036980e0264","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:32:22 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:32:24 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 22:32:24 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244523446487391e0264","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:32:24 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:32:27 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 22:32:28 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244523478957953e0264","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:32:28 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:32:32 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 22:32:32 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 22:32:40 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244523605361551e0264","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:32:40 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:32:41 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 22:32:41 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244523617821733e0264","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:32:41 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:32:43 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 22:32:44 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244523640232050e0264","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:32:44 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:32:47 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 22:32:47 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244523672762497e0264","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:32:47 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:32:51 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 22:32:51 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 22:34:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803517244524739568664e16bf","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:34:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:34:35 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 22:34:35 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803517244524752268783e16bf","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:34:35 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:34:37 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 22:34:37 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803517244524774958989e16bf","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:34:37 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:34:40 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 22:34:40 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803517244524807671381e16bf","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:34:40 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:34:44 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 22:34:44 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 22:36:13 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/08/01/18/59/17/121081248_p0.png
2024-08-23 22:47:22 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00703417244532422983260e0df1","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:47:22 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:47:23 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 22:47:23 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00703417244532435683422e0df1","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:47:23 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:47:25 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 22:47:26 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244532466292532ed927","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:47:26 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:47:29 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 22:47:30 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244532498973071ed927","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:47:30 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:47:34 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 22:47:34 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 22:49:15 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244533548758657ed927","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:49:15 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:49:16 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 22:49:16 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244533561188795ed927","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:49:16 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:49:18 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 22:49:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244533583671063ed927","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:49:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:49:21 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 22:49:21 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244533616361354ed927","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 22:49:21 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 22:49:25 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 22:49:25 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:02:47 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702c17244541671463039ed5b4","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:02:47 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:02:48 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:02:48 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702c17244541684103147ed5b4","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:02:48 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:02:50 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:02:50 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702c17244541706793431ed5b4","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:02:50 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:02:53 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:02:54 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702c17244541739703904ed5b4","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:02:54 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:02:58 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:02:58 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:08:04 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702d17244544839954643ee794","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:08:04 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:08:05 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:08:05 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702d17244544852384775ee794","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:08:05 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:08:07 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:08:07 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702d17244544874895049ee794","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:08:07 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:08:10 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:08:10 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702d17244544907375366ee794","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:08:10 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:08:14 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:08:14 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:09:06 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701617244545460047632e026e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:09:06 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:09:07 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:09:07 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701617244545472747782e026e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:09:07 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:09:09 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:09:09 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701617244545495388033e026e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:09:09 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:09:12 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:09:12 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701617244545528288365e026e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:09:12 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:09:16 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:09:16 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:10:12 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701617244546119277000e026e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:10:12 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:10:13 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:10:13 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701617244546132477175e026e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:10:13 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:10:15 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:10:15 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701617244546155237396e026e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:10:15 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:10:18 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:10:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701617244546187917759e026e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:10:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:10:22 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:10:22 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:21:42 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244553025494675e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:21:42 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:21:43 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:21:43 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244553038314820e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:21:43 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:21:45 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:21:46 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244553060995103e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:21:46 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:21:49 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:21:49 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244553093715513e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:21:49 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:21:53 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:21:53 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:22:56 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244553767025400e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:22:56 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:22:57 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:22:58 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244553779695486e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:22:58 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:23:00 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:23:00 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244553802385673e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:23:00 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:23:03 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:23:03 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701517244553835185970e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:23:03 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:23:07 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:23:07 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:24:20 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803017244554603852801efd51","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:24:20 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:24:21 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:24:21 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803017244554616242950efd51","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:24:21 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:24:23 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:24:24 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803017244554638793286efd51","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:24:24 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:24:27 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:24:27 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803017244554671253719efd51","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:24:27 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:24:31 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:24:31 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:26:21 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803717244555817407225eec35","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:26:21 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:26:22 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:26:23 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803717244555830067452eec35","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:26:23 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:26:25 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:26:25 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803717244555852697763eec35","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:26:25 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:26:28 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:26:28 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803717244555885378215eec35","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:26:28 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:26:32 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:26:32 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:27:08 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/07/10/01/39/13/120305859_p0.jpg
2024-08-23 23:43:40 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702e17244566206128070ee570","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:43:40 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:43:41 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:43:42 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702e17244566218578251ee570","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:43:42 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:43:44 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:43:44 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702e17244566241218653ee570","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:43:44 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:43:47 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:43:47 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702e17244566273711108ee570","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:43:47 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:43:51 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:43:51 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:45:55 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2022/05/22/00/00/13/98508189_p0.jpg
2024-08-23 23:49:16 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803517244569567487858e16be","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:49:16 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:49:17 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:49:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803517244569579938035e16be","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:49:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:49:20 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:49:20 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803517244569602508414e16be","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:49:20 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:49:23 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:49:23 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803517244569635158972e16be","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:49:23 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:49:27 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:49:27 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:53:25 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244572050335287ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:53:25 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:53:26 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:53:26 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244572062965547ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:53:26 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:53:28 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:53:28 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244572085395907ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:53:28 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:53:31 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:53:31 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803817244572117876328ed934","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:53:31 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:53:35 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:53:35 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-23 23:58:10 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701417244574901923020e027e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:58:10 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:58:11 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-23 23:58:11 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701417244574914573147e027e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:58:11 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:58:13 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-23 23:58:13 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701417244574937183400e027e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:58:13 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:58:16 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-23 23:58:17 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00701417244574969843715e027e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-23 23:58:17 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-23 23:58:21 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-23 23:58:21 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-24 00:00:43 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702f17244576436056296e0125","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:00:43 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:00:44 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-24 00:00:45 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702f17244576448476445e0125","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:00:45 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:00:47 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-24 00:00:47 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702f17244576471186739e0125","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:00:47 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:00:50 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-24 00:00:50 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702f17244576503687110e0125","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:00:50 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:00:54 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-24 00:00:54 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-24 00:02:22 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702f17244577424633130e0125","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:02:22 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:02:23 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-24 00:02:23 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702f17244577437143304e0125","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:02:23 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:02:25 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-24 00:02:26 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702f17244577459673537e0125","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:02:26 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:02:29 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-24 00:02:29 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00702f17244577492373911e0125","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:02:29 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:02:33 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-24 00:02:33 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-24 00:04:38 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00703317244578779681295e1b85","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:04:38 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:04:39 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-24 00:04:39 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00703317244578792381461e1b85","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:04:39 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:04:41 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-24 00:04:41 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00703317244578815061767e1b85","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:04:41 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:04:44 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-24 00:04:44 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00703317244578847782193e1b85","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:04:44 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:04:48 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-24 00:04:48 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-24 00:10:54 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2024/07/21/00/07/30/120709771_p0.png
2024-08-24 00:11:15 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803317244582756066177e0994","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:11:15 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:11:16 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-24 00:11:17 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803317244582768766392e0994","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:11:17 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:11:19 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-24 00:11:19 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803317244582791496704e0994","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:11:19 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:11:22 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-24 00:11:22 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00803317244582824267352e0994","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:11:22 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:11:26 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-24 00:11:26 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2024-08-24 00:15:49 - WARNING - aliyundrive.py - 831 - Token is about to expire, try to auto refresh.
2024-08-24 00:19:47 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2021/02/14/00/00/10/87745579_p0.png
2024-08-24 00:23:27 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800117244590076913029e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:23:27 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:23:28 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2024-08-24 00:23:29 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800117244590089343159e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:23:29 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:23:31 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2024-08-24 00:23:31 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800117244590111733457e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:23:31 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:23:34 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2024-08-24 00:23:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800117244590144193795e0262","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|\\"}
2024-08-24 00:23:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2024-08-24 00:23:38 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2024-08-24 00:23:38 - ERROR - aliyundrive.py - 966 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
